faunadb-java-dsl/src/main/java/com/faunadb/client/types/Value.java

```diff
@@ -151,19 +177,50 @@ private Value() {
     return field.get(this).get();
   }
 
+  /**
+   * Attempts to decode the value using the reflection {@link Decoder}.
+   *
+   * @param <T> the type to convert to
+   * @param clazz a class type to convert
+   * @return the converted result, or null if conversion failed
+   * @see Decoder
+   */
+  public final <T> T get(Class<T> clazz) {
+    return to(clazz).get();
+  }
+
   /**
    * Safely attempts to extract a {@link Field} from this value.
    *
    * @param <T> the type of returned field
    * @param field {@link Field} to extract
    * @return An {@link Optional} containing the resulting value, if the field extraction was successful.
-   * It returns {@link Optional#absent()}, otherwise.
+   * It returns {@link Optional#empty()}, otherwise.
    * @see Field
    */
   public final <T> Optional<T> getOptional(Field<T> field) {
     return field.get(this).getOptional();
   }
 
+  /**
+   * Converts this Value into an {@link Optional}, where {@link NullV}
```

Comment:
We're trying to not mention the value types in the docs as it made people confused before.